src/fleet/azext_fleet/_help.py

```diff
@@ -0,0 +1,96 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['fleet'] = """
+    type: group
+    short-summary: Commands to manage fleet.
+"""
+
+helps['fleet create'] = """
+    type: command
+    short-summary: Create a new fleet.
+    parameters:
+        - name: --tags
+          type: string
+          short-summary: The tags of the managed cluster. The managed cluster instance and all resources managed by the cloud provider will be tagged.
+        - name: --dns-name-prefix -p
+          type: string
+          short-summary: Prefix for hostnames that are created. If not specified, generate a hostname using the
+                         managed cluster and resource group names.
+"""
+
+helps['fleet patch'] = """
+    type: command
+    short-summary: Patch an existing fleet.
+    parameters:
+        - name: --tags
+          type: string
+          short-summary: The tags of the managed cluster. The managed cluster instance and all resources managed by the cloud provider will be tagged.
+"""
+
+helps['fleet get'] = """
+    type: command
+    short-summary: Get an existing fleet.
+"""
+
+helps['fleet list-by-resource-group'] = """
+    type: command
+    short-summary: List fleet by resource group.
+"""
+
+helps['fleet list-by-subscription'] = """
+    type: command
+    short-summary: List fleet by subscription id.
+"""
+
+helps['fleet delete'] = """
+    type: command
+    short-summary: Delete an existing fleet.
+"""
+
+helps['fleet get-credentials'] = """
+    type: command
+    short-summary: List fleet kubeconfig credentials
+    parameters:
+    - name: --overwrite-existing
+      type: bool
+      short-summary: Overwrite any existing cluster entry with the same name.
+    - name: --file -f
+      type: string
+      short-summary: Kubernetes configuration file to update. Use "-" to print YAML to stdout instead.
```

Comment:
We already have ` --output -o` argument to support yaml output in stdout.

Reply:
borrowed aks-preview implementation
https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/_help.py#L1622